### PR TITLE
Release v1.3.1

### PR DESCRIPTION
**Pull request by omenUchiha**
## Description

**Features**
- Added responsiveness
- Added error screen
- Updated loader

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx [hunk 1]
```diff
@@ -15,11 +16,16 @@ import {
...
 import { CustomComponents } from "../../types/prop-types/CustomComponents";
+import LMAIChatbotErrorScreen, {
+  LMAIChatbotErrorScreenProps,
+} from "./LMAIChatbotErrorScreen";
 
 export interface LMChatAIButtonProps {
   buttonText?: string;
   loadingScreenAnimatons?: JSON;
   previewText?: string;
+  closeAiChatbot?: () => void;
```
Comment (animeshdas-lm) [inline core/src/components/LMAIChatbot/LMChatAIButton.tsx]:
```suggestion
  closeAIChatbot?: () => void;
```

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx [hunk 2]
```diff
@@ -34,7 +40,20 @@ const LMChatAIButton: React.FC<
   loadingScreenAnimatons,
 }) => {
   const [isAiBotOpen, setIsAiBotOpen] = useState<boolean>(false);
```
Comment (animeshdas-lm) [inline core/src/components/LMAIChatbot/LMChatAIButton.tsx]:
```suggestion
  const [isAIBotOpen, setIsAIBotOpen] = useState<boolean>(false);
```

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx [hunk 3]
```diff
@@ -34,7 +40,20 @@ const LMChatAIButton: React.FC<
...
+      const timeoutId = setTimeout(() => {
+        setShowSettingUpChatbotText(true);
+      }, 3000);
+      return () => {
+        clearTimeout(timeoutId);
+      };
+    }
+  }, [isAiBotOpen]);
   const openAIBot = () => {
     setIsAiBotOpen(true);
```
Comment (animeshdas-lm) [inline core/src/components/LMAIChatbot/LMChatAIButton.tsx]:
```suggestion
    setIsAIBotOpen(true);
```

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx (hunk 1 above)
Comment (omenUchiha) [reply core/src/components/LMAIChatbot/LMChatAIButton.tsx]: Done

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx [hunk 4]
```diff
@@ -107,6 +150,8 @@ const LMChatAIButton: React.FC<
             <LMAIChatbot
               previewText={previewText}
               loadingScreenAnimatons={loadingScreenAnimatons}
+              closeAiChatbot={closeAIBot}
```
Comment (animeshdas-lm) [inline core/src/components/LMAIChatbot/LMChatAIButton.tsx]:
```suggestion
              closeAIChatbot={closeAIBot}
```

File: core/src/components/LMAIChatbot/LMChatAiChatbotHeader.tsx [hunk 5]
```diff
@@ -5,8 +5,15 @@ import { ChatroomTypes } from "../../enums/lm-chatroom-types";
...
 import { Utils } from "../../utils/helpers";
+import CloseIconImage from "../../assets/img/close-chatbot-mobile-icon.png";
 
-const LMChatAIChatbotHeader: FC = () => {
+export interface LMChatAIChatbotHeaderInterface {
+  closeAiChatbot?: () => void;
+}
+
+const LMChatAIChatbotHeader: FC<LMChatAIChatbotHeaderInterface> = ({
+  closeAiChatbot,
```
Comment (animeshdas-lm) [inline core/src/components/LMAIChatbot/LMChatAiChatbotHeader.tsx]:
```suggestion
  closeAIChatbot,
```

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx (hunk 2 above)
Comment (omenUchiha) [reply core/src/components/LMAIChatbot/LMChatAIButton.tsx]: done

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx (hunk 3 above)
Comment (omenUchiha) [reply core/src/components/LMAIChatbot/LMChatAIButton.tsx]: done

**Review by animeshdas-lm — CHANGES_REQUESTED** [review]
please do the requested changes

File: core/src/components/LMAIChatbot/LMChatAIButton.tsx (hunk 4 above)
Comment (omenUchiha) [reply core/src/components/LMAIChatbot/LMChatAIButton.tsx]: done

File: core/src/components/LMAIChatbot/LMChatAiChatbotHeader.tsx (hunk 5 above)
Comment (omenUchiha) [reply core/src/components/LMAIChatbot/LMChatAiChatbotHeader.tsx]: done

**Review by animeshdas-lm — APPROVED** [review]

**Pull request merged**